### ISSUE TITLE
Error - No module named docopt

### ISSUE THREAD

**Issue by schmkr** [deanishe/alfred-fakeum#5]
Hi,

I just downloaded (and then updated) the latest release of this workflow. When I try to run `fakeconfig` I run into this error:

```
Starting debug for 'Fakeum'

[2017-12-02 23:16:01][ERROR: input.scriptfilter] Code 1: 23:16:01 workflow.py:2247 DEBUG    ---------- Fakeum (2.0.0) ----------
23:16:01 workflow.py:1657 DEBUG    reading settings from /Users/a/Library/Application Support/Alfred 3/Workflow Data/net.deanishe.alfred-fakeum/settings.json
23:16:01 workflow.py:2531 DEBUG    update check not due
23:16:01 workflow.py:2266 ERROR    No module named docopt
Traceback (most recent call last):
  File "/Users/a/Library/Application Support/Alfred 3/Alfred.alfredpreferences/workflows/user.workflow.6C513BC9-3DA6-49CD-A916-8DB68669AC84/workflow/workflow.py", line 2259, in run
    func(self)
  File "config.py", line 191, in main
    from docopt import docopt
ImportError: No module named docopt
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/handlers.py", line 76, in emit
    if self.shouldRollover(record):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/handlers.py", line 156, in shouldRollover
    msg = "%s\n" % self.format(record)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 732, in format
    return fmt.format(record)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 471, in format
    record.message = record.getMessage()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 335, in getMessage
    msg = msg % self.args
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 46: ordinal not in range(128)
Logged from file workflow.py, line 2268
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 859, in emit
    msg = self.format(record)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 732, in format
    return fmt.format(record)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 471, in format
    record.message = record.getMessage()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 335, in getMessage
    msg = msg % self.args
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 46: ordinal not in range(128)
Logged from file workflow.py, line 2268
23:16:01 workflow.py:2289 DEBUG    ---------- finished in 0.052s ----------
```

Any idea how this is possible?

**Comment by maxim-uvarov** [comment]
same problem

**Comment by maxim-uvarov** [comment]
Well, I found that command in a terminal can fix it:
`pip install docopt`

**Comment by deanishe** [comment]
Fixed in [v2.0.1](https://github.com/deanishe/alfred-fakeum/releases/tag/v2.0.1).

**Comment by schmkr** [comment]
Yep, works like a charm now! Thanks.
